projects/vdk-core/plugins/vdk-logging-json/src/vdk/plugin/logging_json/logging_json.py

```diff
@@ -1,56 +1,74 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+import json
 import logging
 from typing import List
 
+from ecs_logging import StdlibFormatter
 from vdk.api.plugin.hook_markers import hookimpl
 from vdk.api.plugin.plugin_registry import IPluginRegistry
 from vdk.internal.builtin_plugins.run.job_context import JobContext
 from vdk.internal.core.statestore import CommonStoreKeys
 
-# the labels follow the labelling recommendations found here: http://ltsv.org/
-format_template = (
-    '"@timestamp":"%(asctime)s","created":"%(created)f","jobname":"{job_name}",'
-    '"level":"%(levelname)s","modulename":"%(name)s","filename":"%(filename)s",'
-    '"lineno":"%(lineno)s","funcname":"%(funcName)s","attemptid":"{attempt_id}","message":"%(message)s"'
-)
 
-
-# this class serves to escape newline characters in the log message
-# as according to https://stackoverflow.com/questions/42068/how-do-i-handle-newlines-in-json
-# it is currently experimental and might be removed
-class EscapeNewlinesAndQuotesFormatter(logging.Formatter):
-    def __init__(self, fmt):
-        super().__init__(fmt=fmt)
-        self.default_time_format = "%Y-%m-%dT%H:%M:%S"
-        self.default_msec_format = "%s.%03dZ"
+class EcsJsonFormatter(StdlibFormatter):
+    def __init__(self, job_name, attempt_id, *args, **kwargs):
+        super().__init__(self, *args, **kwargs)
+        self.job_name = job_name
+        self.attempt_id = attempt_id
```

Comment:
nit: consider making private 

```suggestion
        self.__job_name = job_name
        self.__attempt_id = attempt_id
```